core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/UnixProcessManager.java

```diff
@@ -53,11 +57,17 @@
                 .getDeclaredField("pid");
         pidField.setAccessible(true);
       } catch (Exception e) {
```

Comment:
I would say yes.
Also on java9 it will print tons of logs.

In case the exception is not displayed there, we will get another exception later saying that the way of getting PID is not managed by the system